slides.typ

```diff
@@ -94,74 +178,33 @@
 #let slides(
     title: none,
     author: none,
+    subtitle: none,
     short-title: none,
     short-author: none,
     date: none,
-    color: teal,
-    document_body
+    theme: slides-default-theme(),
+    body
 ) = {
-    show heading.where(level: 1): h => {
-        section.update(h.body)
-    }
-
-    show heading.where(level: 2): h => {
-        pagebreak(weak: true)
-        logical-slide.step()
-        h
-    }
-
     set text(
         size: 25pt,
     )
 
-    let decoration(position, body) = {
-        let border = 1mm + color
-        let strokes = (
-            header: ( bottom: border ),
-            footer: ( top: border )
-        )
-        block(
-            stroke: strokes.at(position),
-            width: 100%,
-            inset: .3em,
-            text(.5em, body)
-        )
-    }
-
     set page(
         paper: "presentation-16-9",
-        margin: ( x: 1em, y: 4em ),
-        header: locate( loc => {
-            if counter(page).at(loc).first() > 1 {
-                decoration("header", section.at(loc))
-            }
-        } ),
-        header-ascent: 3em,
-        footer: locate( loc => {
-            if counter(page).at(loc).first() > 1 {
-                decoration("footer")[
-                    #short-author #h(10fr)
-                    #short-title #h(1fr)
-                    #date #h(10fr)
-                    #logical-slide.display()
-                ]
-            }
-        } ),
-        footer-descent: 3em,
+        margin: 0pt,
```

Comment:
This should be configurable too I guess, even though it is generally unwanted to do so (but power to the user)

Reply:
I think this would not only be unwanted but it kinda defeats the purpose of themes, as I was imagining to give them the promise "the slide is yours, use every pixel to your delight; if it looks bad, it's your fault". So _if_ the user wanted different margins, that should be controlled via the theme. I can imagine to make the default theme accept a margin parameter.

Reply:
Sure then, I agree it might be too niche. Themes probably should indeed control that